src/data/reducers/assets.js

```diff
@@ -34,6 +59,33 @@ const list = (state = [], action) => {
   }
 };
 
+const listMetaData = (state = metaDataInitial, action) => {
+  switch (action.type) {
+    case assetActions.REQUEST_ASSETS_SUCCESS:
+      return {
+        start: action.data.start,
+        end: action.data.end,
+        page: action.data.page,
+        pageSize: action.data.pageSize,
+        totalCount: action.data.totalCount,
+      };
+    default:
+      return state;
+  }
+};
+
+const sortMetaData = (state = sortOrderInitial, action) => {
```

Comment:
sortMetaData is a slice of the Redux state that represents sort on the assets. It represents the actual, current state of the table's sorting.